### ISSUE TITLE
TypeError: "exports" is read-only

### ISSUE THREAD

**Issue by alexsobolenko** [Trekels/vue-trans#3]
I use this package like in guide:
```
import Vue from 'vue';
import transFilter from 'vue-trans';
Vue.use(transFilter);
```
but I see error in console:
http://prntscr.com/pg3pep

**Comment by sanderquirynen** [comment]
I'm getting the same error.

**Comment by Trekels** [comment]
@alexsobolenko did you find a solution for this issue ?

**Comment by jeroendk** [comment]
@Trekels Getting the same error, you might want to look into this answer:
https://github.com/webpack/webpack/issues/4039#issuecomment-273804003

"You can't mix import and module.exports."

**Comment by Trekels** [comment]
@jeroendk I changed the export statement and published v0.1.1 this should resolve the issue.

**Comment by jeroendk** [comment]
@Trekels Thanks, tested it and it works now!
